### PR TITLE
feat: ability to reorder entries

### LINKED ISSUE #19: Provide a way to arrange/drag the entries
This comes pretty handy since TokenVault entries are arranged in the way they are entered and there is no current way to sort them.

Something like this:

image

> The _hamburguer_ icon serves that purpose (to drag/reorder the entries) in that case.

### PR DESCRIPTION
**Pull request by johnny243**
- [x] verify user experience in mobile
- [x] add option to reorder by `service` (ascending or descending)
- [x] show handle that indicates the entries can dragged
- [x] verify theme support

https://user-images.githubusercontent.com/5891646/119704807-11b45e00-be26-11eb-9714-34af2a54a8d6.mp4

closes #19

### CODE REVIEW

**Comment by mobitar** [comment]
👏👏

We can change alert copy to maybe "Are you sure you want to auto-sort all entries alphabetically based on service name?" or something similar.

**Comment by mobitar** [comment]
How is this handled on mobile btw?

File: app/components/ViewEntries.js [hunk 1]
```diff
@@ -1,29 +1,79 @@
...
+  const result = Array.from(list);
+  const [removed] = result.splice(startIndex, 1);
+  result.splice(endIndex, 0, removed);
+
+  return result;
+};
+
+const ViewEntries = ({ entries, onEdit, onRemove, onCopyToken, canEdit, updateEntries }) => {
+  const onDragEnd = (result) => {
+    // dropped outside the list
```
Comment (mobitar) [inline app/components/ViewEntries.js:16]:
```suggestion
    const droppedOutsideList = !result.destination;
```

**Review by mobitar — APPROVED** [review]

**Comment by johnny243** [comment]
> How is this handled on mobile btw?

On mobile, you need to long press to drag-n-drop an entry.

**Pull request merged**